app/presenters/check_answers/defendant_details_card.rb

```diff
@@ -0,0 +1,67 @@
+module CheckAnswers
+  class DefendantDetailsCard < Base
+    attr_reader :defendant_details
+
+    KEY = 'defendant_summary'.freeze
+
+    def initialize(claim)
+      @defendant_details = claim.defendants
+    end
+
+    def route_path
+      KEY
+    end
+
+    def title
+      I18n.t('steps.check_answers.groups.about_defendant.defendant_details.title')
+    end
+
+    def rows
+      all_rows = []
+      all_rows.concat main_defendant_rows
+      additional_defendants.each_with_index do |defendant, index|
+        all_rows.concat additional_defendant_rows(defendant, index + 1)
+      end
+      all_rows
+    end
```

Comment:
suggestion
```ruby
main_defendant_rows + 
  additional_defendants.map.with_index do |defendant, index|
    additional_defendant_rows(defendant, index + 1)
  end
```

Reply:
tried doing something like this but it kept breaking the page, couldn't immediately figure out why so moved on in the interest of time to refactor later